ecs-init/docker/docker.go

```diff
@@ -221,6 +222,7 @@ func (c *Client) getContainerConfig() *godocker.Config {
 		"ECS_AVAILABLE_LOGGING_DRIVERS":         `["json-file","syslog","awslogs","none"]`,
 		"ECS_ENABLE_TASK_IAM_ROLE":              "true",
 		"ECS_ENABLE_TASK_IAM_ROLE_NETWORK_HOST": "true",
+		"ECS_CONTAINER_LABELS":                  "",
```

Comment:
how about `ECS_AGENT_LABELS` since these labels are only for the agent container itself rather than all the containers managed by ECS